compiler/q-implant-op-level-test/import/Add_000/__init__.py

```diff
@@ -0,0 +1,38 @@
+from test_utils import TestCase
+from test_utils import gen_random_tensor
+
+
+class Add_000_Q8(TestCase):
+    def __init__(self):
+        self.name = _name_
+
+    def generate(self) -> dict:
+        json_content = dict()
+
+        # Generate ifm1
+        json_content['ifm1'] = gen_random_tensor(
+            "uint8",  # dtype_str
+            (1),  # scale_shape
+            (1),  # zerop_shape
+            0)  # quantized_dimension
+
+        # Generate ifm2
+        json_content['ifm2'] = gen_random_tensor(
+            "uint8",  #dtype_str
+            (1),  # scale_shape
+            (1),  # zerop_shape
+            0)  # quantized_dimension
+
+        # Generate ofm
+        json_content['ofm'] = gen_random_tensor(
+            "uint8",  # dtype_str
+            (1),  # scale_shape
+            (1),  # zerop_shape
+            0)  # quantized_dimension
+
+        return json_content
+
+
+_name_ = 'Add_000_Q8'
+
+_model_ = Add_000_Q8()
```

Comment:
variable name is `_model_` but how is `Add_000_Q8()` related to a model?

Reply:
where is _name_ and _model_ used ?

Reply:
The reason why named model is because that have information(scale, zero point, shape, ...) of model.

Reply:
> ... have information ...

then please another word than `model` to express `information of a model`

Reply:
Can i use `_info_` instead of '_model_'?

Reply:
How about `_test_case_` ? `class Add_000_Q8` inherits `TestCase`.

Reply:
> How about _test_case_

looks good to me :)